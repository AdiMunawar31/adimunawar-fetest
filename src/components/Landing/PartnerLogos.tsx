import React from "react";

const PartnerLogos = () => {
  return (
    <>
      <div className='text-gray-600 font-bold'>
        <span>Our Partnership :</span>
      </div>
      <div className='flex gap-6 lg:gap-12 justify-between grayscale lg:w-2/3'>
        <img
          src='https://tailus.io/sources/blocks/tech-startup/preview/images/clients/airbnb.svg'
          className='h-8 sm:h-10 w-auto lg:h-12'
          alt='airbnb'
        />
        <img
          src='https://tailus.io/sources/blocks/tech-startup/preview/images/clients/ge.svg'
          className='h-8 sm:h-10 w-auto lg:h-12'
          alt='ge'
        />
        <img
          src='https://tailus.io/sources/blocks/tech-startup/preview/images/clients/coty.svg'
          className='h-8 sm:h-10 w-auto lg:h-12'
          alt='coty'
        />
        <img
          src='https://tailus.io/sources/blocks/tech-startup/preview/images/clients/microsoft.svg'
          className='h-8 sm:h-10 w-auto lg:h-12'
          alt='microsoft'
        />
      </div>
    </>
  );
};

export default PartnerLogos;
