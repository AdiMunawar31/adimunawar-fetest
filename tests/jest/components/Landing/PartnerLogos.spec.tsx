import React from "react";
import renderer from "react-test-renderer";
import PartnerLogos from "@components/Landing/PartnerLogos";

it("renders PartnerLogos correctly", () => {
  const tree = renderer.create(<PartnerLogos />).toJSON();
  expect(tree).toMatchSnapshot();
});
