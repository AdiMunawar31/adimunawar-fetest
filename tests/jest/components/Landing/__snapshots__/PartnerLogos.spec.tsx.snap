// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PartnerLogos correctly 1`] = `
[
  <div
    className="text-gray-600 font-bold"
  >
    <span>
      Our Partnership :
    </span>
  </div>,
  <div
    className="flex gap-6 lg:gap-12 justify-between grayscale lg:w-2/3"
  >
    <img
      alt="airbnb"
      className="h-8 sm:h-10 w-auto lg:h-12"
      src="https://tailus.io/sources/blocks/tech-startup/preview/images/clients/airbnb.svg"
    />
    <img
      alt="ge"
      className="h-8 sm:h-10 w-auto lg:h-12"
      src="https://tailus.io/sources/blocks/tech-startup/preview/images/clients/ge.svg"
    />
    <img
      alt="coty"
      className="h-8 sm:h-10 w-auto lg:h-12"
      src="https://tailus.io/sources/blocks/tech-startup/preview/images/clients/coty.svg"
    />
    <img
      alt="microsoft"
      className="h-8 sm:h-10 w-auto lg:h-12"
      src="https://tailus.io/sources/blocks/tech-startup/preview/images/clients/microsoft.svg"
    />
  </div>,
]
`;
